docs/release.md

```diff
@@ -20,313 +23,242 @@ release, those are:
-```
+The stabilization phase of a release begins with the stable branch creation.
+The stabilization of a release takes place on the stable branch of the release.
 
-**Example: Creating a new rc v0.31.0-rc.0**
-```
-hack/release.sh --new-release v0.31.0-rc.0 --dry-run=false
-```
+### Bug fixes
 
-**Example: Promoting a release candidate v0.31.0-rc-1 to official v0.30.0 release.**
-```
-hack/release.sh --promote-release-candidate v0.31.0-rc-1 --dry-run=false
-```
+Once the branch is created, only backports will be allowed into the stable branch, following KubeVirt's [backport policy](release-branch-backporting.md).
 
-**Example: Creating a patch release v0.31.1. The branch will automatically be detected.**
-```
-hack/release.sh --new-release v0.31.1 --dry-run=false
-```
+### Blockers
 
-## Creating New Minor Releases
-The release process is mostly automatic and consists of the following steps:
+If blockers are detected, then they first have to be fixed, and then a new release candidate is generated and will be promoted after giving the impacted parties enough time to validate the blocker is addressed.
 
-1. Create the branch and initial RC.
+The procedure for handling blockers is described in [release-procedure.md](release-procedure.md).
 
-   ```hack/release.sh --new-branch $RELEASE_BRANCH --new-tag ${TAG}.rc.0```
+### Release readiness
 
-2. Wait 5 business days
+The following pre-conditions need to be met before any release tags are created (RC, GA).
 
-3. Promote RC to official release if no blockers exist.
+#### CI Provider
 
-   ```hack/release.sh --promote-release-candidate ${TAG}.rc.0```
+A pre-condition for the GA of a KubeVirt release is the presence of a KubeVirt CI provider for the corresponding Kubernetes release.
+Around the beta release of the corresponding Kubernetes release, the KubeVirt team will start to work on a KubeVirt CI provider.
+The assumption is, that the KubeVirt CI provider will be available by KubeVirt's enhancement freeze.
 
-4. Wait for [travis](https://travis-ci.org/kubevirt/kubevirt/) to finish, and
-   check that the binary artifacts got attached to the release at
-   `https://github.com/kubevirt/kubevirt/releases/tag/${TAG}`
-   and that the containers were correctly tagged and pushed to
-   <https://hub.docker.com/r/kubevirt/>
+The introduction of a new provider has the following phases:
+1. Provider is created
+2. Provider is published
+3. Provider lands in kubevirt/kubevirt
+4. Periodic and presubmit lanes get introduced for the sigs `compute`, `network`, `storage` and `operator`, where
+   1. while the periodics deliver a signal for how KubeVirt and the provider are interacting, the presubmits initially are to be triggered manually, so that teams can work on fixing bugs in either the provider or the KubeVirt code
+   2. at the point when the periodics look stable enough, the presubmits are turned on to run on every PR
+   3. if the signal is looking stable enough, they are made voting
```

Comment:
not sure what is meant by "they are made voting"

Reply:
will gate a PR